src/suit/js/SUIT.js

```diff
@@ -38,6 +39,9 @@ const lsstEntry= (label, url) => (
     {
         label,
         value: url,
+        fovDeg: 10,
+        centerWP: makeWorldPt(62,-37).toString(),
+        hipsUrl: 'http://hips.hips.svc.cluster.local:8080/api/hips/images/color_gri',
```

Comment:
Let's put in the correct endpoint for production, https://data.lsst.cloud/api/hips/images/color_gri , and then override it with the `hips.hips.svc.cluster.local:8080` kludge in deployment.  That will make this more robust and less likely to confuse someone in the future.